app/services/ClaimToAdjustHelper.scala

```diff
@@ -43,11 +43,11 @@ trait ClaimToAdjustHelper {
 
   private val isUnpaidPoAOne: DocumentDetail => Boolean = documentDetail =>
     documentDetail.documentDescription.contains(POA1) &&
-      !documentDetail.outstandingAmount.contains(BigDecimal(0))
+      !(documentDetail.outstandingAmount == (0))
```

Comment:
we can make this easier to read if we can apply next change:
(documentDetail.outstandingAmount != 0)